MP3 Tag Editor/Mp3TagsManager.cs

```diff
@@ -91,6 +93,57 @@ private static void ModifyMp3Tag(TagLib.File file, Mp3Tag mp3Tag, dynamic value)
             HandlePropertyModificationException(e, mp3Tag, value);
         }
     }
+    
+    /// <summary>
+    /// Modifies the specified MP3 tags using a CSV file.
+    /// It is assumed the MP3 files are in the same order as the CSV format.
+    /// </summary>
+    /// <param name="files">The list of MP3 files to modify.</param>
+    /// <param name="sourceProperties">The properties to modify as given by the CSV file</param>
+    /// <param name="sourceValues">A 2D array consisting of an array of id values per song as given by the CSV file</param>
+    public static void ModifyMp3TagsWithCsv(IEnumerable<TagLib.File> files,IEnumerable<string> sourceProperties,IEnumerable<string[]> sourceValues)
+    {
+        var mp3Files = files.ToList();
+        var idValues = sourceValues.ToArray();
+        var i = 0;
+        mp3Files.ForEach(file => ModifyMp3TagWithCsv(file, sourceProperties.ToArray(), idValues[i++]));
+    }
+
+    /// <summary>
+    /// Assigns the values from a CSV file to the MP3 tags of a TagLib.File.
+    /// </summary>
+    /// <param name="file">The TagLib.File object to modify.</param>
+    /// <param name="sourceProperties">The properties to modify as given by the CSV file</param>
+    /// <param name="sourceValues">The values to use as given by the CSV file</param>
+    private static void ModifyMp3TagWithCsv(TagLib.File file, string[] sourceProperties,string[] sourceValues)
+    {
+        foreach (var property in sourceProperties)
+        {
+            ModifyMp3Tags(new [] {file}, property, sourceValues[Array.IndexOf(sourceProperties, property)]);
+        }
+    }
+
+    /// <summary>
+    /// Modifies the specified MP3 tags using a an XML.
+    /// It is assumed the MP3 files are in the same order as the XML format.
+    /// </summary>
+    /// <param name="files">The list of MP3 files to modify.</param>
+    /// <param name="sourceXml">The XML file to use.</param>
+    public static void ModifyMp3TagsWithXml(IEnumerable<TagLib.File> files, XmlDocument sourceXml)
+    {
+        var mp3Files = files.ToArray();
+        var xmlRoot = sourceXml.DocumentElement;
+        var i = 0;
+
+        foreach (XmlNode track in xmlRoot.ChildNodes)
+        {
+            foreach (XmlNode property in track.ChildNodes)
+            {
+                ModifyMp3Tags(new [] {mp3Files[i]}, property.Name, property.InnerText);
```

Comment:
Consider changing this to the following instead. I personally prefer this over creating a new array.
```                
		Enum.TryParse(property.Name, out Mp3Tag tag);
                ModifyMp3Tag(mp3Files[i], tag, property.InnerText);
```

Reply:
Done, but be wary this would make the acceptable formats a lot stricter, I think it would be better if the functionality in ModifyMp3Tags would made into a general parsing method that would also handle things like the Artist/Artists situation.

Reply:
I'll work on it with the already opened issue/enhancmenet